.ci/deploy/pantheon/dev-multidev

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -eo pipefail
+
+#
+# This script deploys the build artifact to Pantheon.
+# On the master branch the dev environment is used.
+# Otherwise a multidev environment is used.
+#
+
+# Authenticate with Terminus
+terminus -n auth:login --machine-token="$TERMINUS_TOKEN"
+
+if [[ $CI_BRANCH != "master" ]]
+then
+    # Create a new multidev environment (or push to an existing one)
+    terminus -n build:env:create "$TERMINUS_SITE.dev" "$TERMINUS_ENV" --yes
```

Comment:
This should have --clone-content also?

Reply:
We can give it a shot but there is [a prior job that runs `multidev:create`](https://github.com/pantheon-systems/example-drops-8-composer/pull/255/files#diff-2bb1205325a28694f90c34db9aa1b73fR15) so the multidev environment is already created by the time we get to this step. During multidev creation content is cloned from the source environment (`dev`).

I used `build:env:create` just in case, as it will create the multidev if it does not already exist but here acts more like `build:env:push`.

I'm not sure if the addition of `--clone-content` means that content cloning will happen on each CI build. If so, it probably isn't desirable as all previous changes to that multidev, such as configuration, will be lost when the database is overwritten during the clone.